src/index.test.ts

```diff
@@ -104,7 +104,7 @@ describe('createStaleWhileRevalidateCache', () => {
       })
       expect(fn).toHaveBeenCalledTimes(1)
       expect(customSerialize).toHaveBeenCalledTimes(1)
-      expect(customDeserialize).toHaveBeenCalledTimes(1)
+      expect(customDeserialize).toHaveBeenCalledTimes(0)
```

Comment:
Do you mind updating this test's description to `should invoke custom serializer and deserializer methods when reading from cache`?

And then to be clear, can you invoke the `swr` function again to assert that the `customDeserialize` is called when the value is now in the cache? Basically, duplicate from line 93 onwards after this set of assertions.